cmd/stackdriver-prometheus-sidecar/statusz.go

```diff
@@ -25,9 +25,137 @@ import (
 	"github.com/prometheus/common/version"
 )
 
+const(
+     statuszHtml = `	
```

Comment:
I don't think it's a good idea to embed a large HTML file in Go source code. It makes it hard to do work with both, and to validate the correctness of the HTML template.

It'd be great if you could find a solution that works in a container and when the binary is installed locally. Currently, we only support the container version, and that works well.

Options to fix this include:
- Modify the Makefile to install the HTML file.
- Embed the data file, e.g. with https://github.com/golang/tools/blob/master/godoc/static/makestatic.go or https://github.com/rakyll/statik

Reply:
Already commented in the main conversation, but moving here for more visibility:
> Maybe [this](https://stackoverflow.com/a/29500100)?

Reply:
jkohen@ let me know your thoughts on igorpeshansky suggestion. We can use "go generate" to add to install the html file in the bin folder where the stackdriver binary is

Reply:
@fifiug `go generate` by itself isn't sufficient, we still need a code generator (script/includetxt.go in that thread) to generate the TOC with embedded data (textfiles.go in that thread). Instead of writing our own generator, we can use `statik`. See https://github.com/rakyll/statik/tree/master/example

Reply:
@jkohen  used statik as code generator. Example is similar to  https://github.com/rakyll/statik/tree/master/example , could you please review my most recent commit ?